src/compiler/utilities.ts

```diff
@@ -2155,6 +1676,475 @@ namespace ts {
         return options.target === ScriptTarget.ES6 ? "lib.es6.d.ts" : "lib.d.ts";
     }
 
+    // Gets the nearest enclosing block scope container that has the provided node
+    // as a descendant, that is not the provided node.
+    export function getEnclosingBlockScopeContainer(node: Node): Node {

```

Comment:
These are still in `utilities.ts`, so was there any reason they were moved around other than locality?